RadixWallet/Features/SplashFeature/Splash.swift

```diff
@@ -152,17 +159,35 @@ public struct Splash: Sendable, FeatureReducer {
 			}
 
 			return delegateCompleted(context: state.context)
+
+		case .showAppLockMessage:
+			state.destination = .errorAlert(.init(
+				title: .init("Warning"),
+				message: .init("To protect your app, use the App Lock system feature"),
+				buttons: [
```

Comment:
minor: we should use `init(title:actions:message:)` since this one is deprecated